Traceback (most recent call last):
  File "/home/vagrant/.pyenv/versions/py3/bin/sirepo", line 33, in <module>
    sys.exit(load_entry_point('sirepo', 'console_scripts', 'sirepo')())
  File "/home/vagrant/src/radiasoft/sirepo/sirepo/sirepo_console.py", line 18, in main
    return pkcli.main("sirepo")
  File "/home/vagrant/src/radiasoft/pykern/pykern/pkcli/__init__.py", line 165, in main
    res = argh.dispatch(parser, argv=argv)
  File "/home/vagrant/.pyenv/versions/3.9.15/envs/py3/lib/python3.9/site-packages/argh/dispatching.py", line 179, in dispatch
    for line in lines:
  File "/home/vagrant/.pyenv/versions/3.9.15/envs/py3/lib/python3.9/site-packages/argh/dispatching.py", line 290, in _execute_command
    for line in result:
  File "/home/vagrant/.pyenv/versions/3.9.15/envs/py3/lib/python3.9/site-packages/argh/dispatching.py", line 273, in _call
    result = function(*positional, **keywords)
  File "/home/vagrant/src/radiasoft/sirepo/sirepo/pkcli/epicsllrf.py", line 38, in run_background
    raise epicsllrf.EpicsDisconnectError("a test EpicsDisconnectError")
sirepo.template.epicsllrf.EpicsDisconnectError: a test EpicsDisconnectError